tests/test_equilibrium.py

```diff
@@ -419,3 +419,16 @@ def test_equilibrium_unused_kwargs():
     with pytest.raises(TypeError):
         _ = Equilibrium(pres=pres, curr=curr)
     _ = Equilibrium(pressure=pres, current=curr)
+
+
+@pytest.mark.slow
+@pytest.mark.unit
+@pytest.mark.solve
+def test_backward_compatible_load_and_resolve():
+    """Test backwards compatibility of load and re-solve."""
+    with pytest.warns(RuntimeWarning):
+        eq = EquilibriaFamily.load(load_from=".//tests//inputs//NCSX_older.h5")[-1]
```

Comment:
minor thing but might want to reduce the resolution a lot after loading to make the compile/solve faster

Reply:
Resolved